packages/big-design/src/components/Modal/spec.tsx

```diff
@@ -263,11 +263,11 @@ test('renders destructive action button', () => {
 
 test('unmounts appropriately', () => {
   const onClick = jest.fn();
-  const { getByTestId, rerender, unmount } = render(<Modal isOpen={true} />);
+  const { getByTestId, unmount } = render(<Modal isOpen={true} />);
 
   unmount();
 
-  rerender(
+  render(
```

Comment:
I was getting this error when rerendering: `Cannot update an unmounted root.`.

Reply:
Interesting, probably because the `button` wasn't the one that was rendered at first 🤷🏼‍♀️ 🤔